numpy/random/tests/test_random.py

```diff
@@ -205,12 +208,13 @@ def test_repeatability(self):
         assert_(tgt[np.dtype(np.bool).name] == res)
 
 
-class TestRandomDist(TestCase):
+class TestRandomDist:
```

Comment:
Because subclasses of TestCase do not support generative tests (http://nose.readthedocs.org/en/latest/writing_tests.html#test-generators)